website/pages/docs/install/glossary.mdx

```diff
@@ -74,3 +74,333 @@ over the internet or wide area network.
 
 Remote Procedure Call. This is a request / response mechanism allowing a
 client to make a request of a server.
+
+# Glossary
```

Comment:
Do we need this header? I am looking at the preview in https://deploy-preview-9348--consul-docs-preview.netlify.app/docs/install/glossary and this page already has a header "Consul Glossary"

Reply:
I updated this to "Consul Vocabulary" as these are the terms that are used specifically in the context of the product.  The rest of the terms I included in the list are used in the broader context of the conversation around networking and microservices.